open-craft/ecommerce
# [BB-3801] Send user profile data to the checkout endpoint

This PR adds the changes to send the user's email address (always) and the first name, last name (if available) to the HyperPay checkout API. This is required for passing the risk checks in production.

**Testing instructions**
* Checkout the source branch of this PR and set up ecommerce to use HyperPay as done with the previous HyperPay PR.
* Go through the ecommerce checkout process with a user who has the first name, and the last name fields set in the LMS.
* Verify that these fields are passed to the HyperPay checkout API.
* Verify that the same data is present in the response received from HyperPay after a successful payment, thereby confirming that the data was indeed passed to the API.
